src/agent/debuglet.ts

```diff
@@ -14,38 +14,32 @@
  * limitations under the License.
  */
 
-import {AuthenticationConfig, Common} from '../types/common';
-const common: Common = require('@google-cloud/common');
-
+import * as assert from 'assert';
 import * as crypto from 'crypto';
 import {EventEmitter} from 'events';
 import * as extend from 'extend';
 import * as fs from 'fs';
-
 import * as metadata from 'gcp-metadata';
-
+import * as http from 'http';
 import * as _ from 'lodash';
 import * as path from 'path';
 import * as util from 'util';
-import * as utils from './util/utils';
-import * as http from 'http';
 
-import {Controller} from './controller';
-import {Debuggee, DebuggeeProperties} from '../debuggee';
+import {Debug, PackageInfo} from '../client/stackdriver/debug';
 import {StatusMessage} from '../client/stackdriver/status-message';
+import {Debuggee, DebuggeeProperties} from '../debuggee';
+import {AuthenticationConfig} from '../types/common';
+import * as stackdriver from '../types/stackdriver';
 
 import {defaultConfig} from './config';
+import {DebugAgentConfig, ResolvedDebugAgentConfig} from './config';
+import {Controller} from './controller';
 import * as scanner from './io/scanner';
 import * as SourceMapper from './io/sourcemapper';
+import * as utils from './util/utils';
 import * as debugapi from './v8/debugapi';
-
-import * as assert from 'assert';
-
-import * as stackdriver from '../types/stackdriver';
-import {DebugAgentConfig, ResolvedDebugAgentConfig} from './config';
-import {Debug, PackageInfo} from '../client/stackdriver/debug';
-import {Logger} from '../types/common';
 import {DebugApi} from './v8/debugapi';
+import consoleLogLevel = require('console-log-level');
```

Comment:
It looks like there is an @types/console-log-level package, can we use that?

Reply:
Absolutely! That's why I wrote it.